ROADMAP.md

```diff
@@ -0,0 +1,72 @@
+# Roadmap
+
+This is a roadmap of future features in consideration for `use-axios-client`, along with some rough API designs.
+
+- [Caching](#caching)
+- [Server-side rendering](#server-side-rendering)
+- [Suspense](#suspense)
+
+## Caching
+
+Provide an optional `cachePolicy` field on hooks to allow consumers to selectively optimize network requests. This entry can be keyed by something like `url` + stringified `data`.
+
+We can persist the cache to a singleton that all hooks can read/write from.
+
+### `no-cache` (default)
+
+Always fire network call for `data` (default):
+
+```js
+const { data, error, loading } = useAxios({
+  url: 'https://example/api',
+  cachePolicy: 'no-cache',
```

Comment:
`no-cache`, `in-memory`, `in-memory-first`, `localStorage`, `sessionStorage`.

Should we make these more consistent rather than some being kebab case and other being camel case?

Reply:
For sure - still not sure on how to name some of these but I agree.